joplin/base/migrations/0033_auto_20180927_0500.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.0.8 on 2018-09-27 05:00
```

Comment:
This is the data migration script, I created it using `python manage.py migrate base --empty` to initialize it without anything in it.

Reply:
Can you add this info to the readme -- basically how to create a migration process.